MumsWhoCode.Services.api/Models/Participants/Participant.cs

```diff
@@ -0,0 +1,41 @@
+﻿// -----------------------------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// -----------------------------------------------------------------------
+
+using System;
+using MumsWhoCode.Services.Api.Models.Users;
+
+namespace MumsWhoCode.Services.Api.Models.Participants
+{
+    public class Participant
+    {
+        public Guid Id { get; set; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public enum Gender
+        { 
+            Female,
+            Male,
+            Other 
+        }
+        public string mobile { get; set; }
```

Comment:
The properties name should be captialized

Reply:
resolved